src/discord/index.js

```diff
@@ -134,6 +134,15 @@ export default class Bot extends EventEmitter {
             .catch(reason => message.reply(`Oops! Something went wrong! Error: ${reason}`));
           break;
         }
+
+        case '!skip': {
+          const res = radio.voteSkip(message.author);
+          if (!res) {
```

Comment:
tbqh i think i might’ve said otherwise but we should at least reply with *something* just so the user gets immediate confirmation that the command worked

Reply:
I'll throw in a somewhat generic message for now. I do agree that having a reply is important because I noticed Virani was confused as to whether or not it worked when he did it without the web application up. 

We'll also be doing the setTopic thing but that kind of update is not right in your face confirmation since it's kinda tucked away at the top in small text with a bunch of other stuff. I'd personally like to be able to differentiate between skips and unskips on the message as well. How would you feel about me changing the return value? We could do something like -1, 0, 1 where -1 is the response to trying to skip when not a DJ, 0 is skip and 1 is unskip.

Reply:
another thing you could do is return an object

you could do some `{voteSkipped: false}` for instance, which as a whole is truthy so it passes error check but then you could inspect the property

Reply:
Oh, and otherwise I could just pass nothing for an error?